projet/users/views.py

```diff
@@ -94,8 +103,59 @@ class ListUsers(generics.ListAPIView):
     to add another permissions apart from the IsAuthenticated
     """
 
+    search_fields = ["email", "first_name", "last_name"]
+    filter_backends = (filters.SearchFilter,)
+
     queryset = User.objects.all()
-    serializer_class = UserSerializer
+    serializer_class = UserBulkSerializer
```

Comment:
Changed to the new serializer we created to avoid showing passwords in the endpoints.